20.0/fpm-alpine/entrypoint.sh

```diff
@@ -43,6 +43,22 @@ file_env() {
     unset "$fileVar"
 }
 
+# change uid/gid of www-data
+if [ "$(id -u)" = 0 ]; then
+    CURRENT_UID=$(id -u www-data)
+    CURRENT_GID=$(id -g www-data)
+    if [ -n "${UID+x}" ] && [ "$UID" -ne $CURRENT_UID ]; then
+        echo "Change UID of www-data from $CURRENT_UID to $UID"
+        usermod -u $UID www-data
```

Comment:
Just a heads up - this won't work since alpine doesn't  ship with``usermod`` command by default.

So either ``shadow`` apk package needs to be installed or ``/etc/passwd`` edited using sed or similar (not really robust approach).

```bash
/entrypoint.sh: line 52: usermod: not found
/entrypoint.sh: line 52: usermod: not found
```

Reply:
Yeah, good point. I've tested it only with the debian variant. I would like to use the shadow package because it seems to be the most stable solution. I added a new commit with shadow and now it works on both variants. Thanks.